rails_panel/assets/javascripts/filters.js

```diff
@@ -8,6 +8,9 @@ angular.module('RailsPanel', [])
   }).
   filter('editorify', function() {
     return function(filename, line) {
+      filename = encodeURI(filename);
```

Comment:
There are still few special characters that are not encoded.

Could you use a method from https://stackoverflow.com/a/46441344 to catch all of them?

```js
function fixedEncodeURIComponent(str) {
  return encodeURIComponent(str).replace(/[!'()*]/g, function(c) {
    return '%' + c.charCodeAt(0).toString(16);
  });
}
```